packages/editor/src/lib/primitives/geometry/Geometry2d.ts

```diff
@@ -73,7 +80,6 @@ export abstract class Geometry2d {
 	abstract nearestPoint(point: Vec, filters?: Geometry2dFilters): Vec
```

Comment:
do we need the `isExcludedByFilter` function at all anymore? nix?

Reply:
i see it's used in `Group2d` - should it be specific to `Group2d` / check that it's only used in that context or something?

Reply:
still part of the Geometry2d contract